packages/trace-viewer/src/versions/traceV3.ts

```diff
@@ -97,6 +97,8 @@ export type ResourceOverride = {
 };
 
 export type FrameSnapshot = {
+  // There was no callId in the original, we are intentionally regressing it.
+  callId: string;
```

Comment:
We can turn `snapshotName` into `callId` for old traces, to preserve the highlight.

Reply:
Yes, but given how backwards support worked, I think it is ok.